chia/rpc/wallet_rpc_api.py

```diff
@@ -2365,6 +2367,9 @@ async def did_find_lost_did(self, request: Dict[str, Any]) -> EndpointResult:
                             uint32(index), uint32(1), False
                         )
                         while derivation_record is not None:
+                            # The assert is needed for the updated DerivationRecord to include secp keys
+                            # No cover added for mypy since there is no test case that reaches this code.
+                            assert isinstance(derivation_record.pubkey, G1Element)  # pragma: no cover
```

Comment:
Ignoring coverage is for the very rare case where we should never test code.  It is not a way to silence the coverage warning.

```suggestion
                            # The assert is needed for the updated DerivationRecord to include secp keys
                            assert isinstance(derivation_record.pubkey, G1Element)
```

Reply:
I'll leave it to the PR author whether adding a test is convenient/relevant but, to be clear, this is not a "new" ignore just making an implicit one explicit.  If the original change to test coverage had explicitly marked all areas of the code not currently covered then these types of ignores would not be necessary in the diffs of PRs like this.

Reply:
I probably should have added that yes, we do make coverage exceptions and merge things that aren't fully covered.  This just isn't the way to do that.